lightning_main.cc

```diff
@@ -1,12 +1,16 @@
 #include "config_parser.h"
+#include "lightning_server.h"
 
 #include <iostream>
 
 int main(int argc, char* argv[]) {
   if (argc != 2) {
-    std::cout << "Usage: ./config_parser <path to config file>\n" << std::endl;
+    std::cout << "Usage: lightning <path to config file>\n" << std::endl;
     return 1;
   }
 
+  LightningServer server(argv[1]);
```

Comment:
If we later have more command-line arguments, it may be worth naming each argument as a variable before using them.